sneakers.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.7"
   spec.add_development_dependency "dotenv"
+  spec.add_development_dependency "pry"
```

Comment:
I'd feel better if you `gem install pry` locally and don't require it as a dev dep.